.github/actions/docker-build-v2/action.yml

```diff
@@ -0,0 +1,53 @@
+name: Build and publish image
+
+description: Builds and pushes the Docker image.
+
+inputs:
+  gcloud-project:
+    description: "GCP project to use for the build"
+    required: true
+  gcloud-credentials:
+    description: "Credentials corresponding to the service account (json)"
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: Set env vars
+      shell: bash
+      run: |
+        echo "GCLOUD_PROJECT=${{ inputs.gcloud-project }}" >> $GITHUB_ENV
+        echo "IMAGE_NAME=${{ github.event.repository.name }}" >> $GITHUB_ENV
+        echo "VERSION=$DOCKER_IMAGE_VERSION" >> $GITHUB_ENV
+
+    - name: 'auth'
+      uses: 'google-github-actions/auth@v0'
+      with:
+        credentials_json: "${{ inputs.gcloud-credentials }}"
+
+    - name: Setup Gcloud
+      uses: google-github-actions/setup-gcloud@v0
+      with:
+        version: "270.0.0"
```

Comment:
Is it required to have a version here? If yes, maybe we can go for a newer version?

Reply:
@alexander-fischer yes good catch. It's not mandatory and if not specified will install the latest gcloud SDK version. I would usually like to pin one recent version to be sure to not have it break all of a sudden but in this case, just pushing an image to GCR without any specific conf I think we are safe so elected to remove the parameter.